CMakeLists.txt

```diff
@@ -103,7 +124,7 @@ if(COVERAGE)
   coverage_turn_on()
 endif()
 
-if(ENABLE_WORKBENCH)
+if(ENABLE_WORKBENCH OR MANTID_QT_LIB STREQUAL "BUILD")
```

Comment:
Could we move anything related to a QtBuild into it's own CMake file?

That way our top level CMakelists.txt reads something like:
```
if(ENABLE_WORKBENCH OR MANTID_QT_LIB STREQUAL "BUILD")
   add_file(buildconfig/CMake/MantidQtTargets.cmake)
```

But if it's too much work we can leave it for the future